libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -41,9 +41,7 @@ class MapboxTripSession(
     override var route: DirectionsRoute? = null
         set(value) {
             field = value
-            ioJobController.scope.launch {
-                navigator.setRoute(value)
-            }
+            navigator.setRoute(value)
```

Comment:
@LukasPaczos / @Guardiola31337 : Does the `setRoute` take a long time which may cause ANR?
In the caching voice case, we need to `preCache` the voiceInstruction as soon as the navigation starts. What we get from the `MapboxNativeNavigatorImpl` is null. It's because the navigator is still setting the route in IO thread.
```kotlin
    override fun getVoiceInstruction(index: Int): VoiceInstruction? =
        navigator.getVoiceInstruction(index)
```
Please let me know if there is any concern about removing this coroutine usage.

Reply:
nvm.